prisma/schema.prisma

```diff
@@ -9,171 +8,277 @@ datasource db {
+model Restaurant {
+  id        String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  name      String
+  ownerId   String   @unique @db.Uuid
+  orderId   String?  @unique @db.Uuid
   createdAt DateTime @default(now())
   updatedAt DateTime @default(now())
 
   branches RestaurantBranch[]
-  orders Order[]
-  menus MenuItem[]
+  orders   Order[]
+  menus    MenuItem[]
 
   owner RestaurantOwner @relation(fields: [ownerId], references: [userId])
 }
 
-model RestaurantBranch{
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  address String
-  restaurantId String @db.Uuid
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
-  
+model RestaurantBranch {
+  id           String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  address      String
+  restaurantId String   @db.Uuid
+  createdAt    DateTime @default(now())
+  updatedAt    DateTime @default(now())
+
   branchMenuItems BranchMenuItem[]
-  
-  restaurant Restaurant @relation(fields: [restaurantId],  references: [id])
-}
-
-model MenuItem{
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  nameOfDish String
-  category String
-  description String
-  price Decimal @db.Decimal(12,2)
-  restaurantId String @db.Uuid
-  imageId String @db.Uuid
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
-  
-  orderLines OrderLine[]
+  Storage         Storage[]
+
+  restaurant Restaurant @relation(fields: [restaurantId], references: [id])
+}
+
+model MenuItem {
+  id           String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  nameOfDish   String
+  category     String
+  description  String
+  price        Decimal  @db.Decimal(12, 2)
+  restaurantId String   @db.Uuid
+  imageId      String   @db.Uuid
+  createdAt    DateTime @default(now())
+  updatedAt    DateTime @default(now())
+
+  orderLines         OrderLine[]
   modificationGroups MenuItemModificationGroup[]
-  branchMenuItems BranchMenuItem[]
+  branchMenuItems    BranchMenuItem[]
 
   restaurant Restaurant @relation(fields: [restaurantId], references: [id])
-  image Image @relation(fields: [imageId], references: [id])
+  image      Image      @relation(fields: [imageId], references: [id])
 }
 
-model BranchMenuItem{
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  branchRestaurantId String @db.Uuid
-  menuItemId String @db.Uuid
-  isAvailable Boolean @default(false)
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
+model BranchMenuItem {
+  id                 String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  branchRestaurantId String   @db.Uuid
+  menuItemId         String   @db.Uuid
+  isAvailable        Boolean  @default(false)
+  createdAt          DateTime @default(now())
+  updatedAt          DateTime @default(now())
 
   branchRestaurant RestaurantBranch @relation(fields: [branchRestaurantId], references: [id])
```

Comment:
please change to restaurantBranch, although it was not written by you. Also, fix the type in code as well and try to run it to make sure everything works fine

Reply:
done, this restaurant Branch has not yet used in a code)